pkg/scheduler/framework/runtime/framework.go

```diff
@@ -384,6 +396,86 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, opts ...Opti
 	return f, nil
 }
 
+func getScoreWeights(f *frameworkImpl, totalPriority int64, plugins []config.Plugin) (error, int64) {
+	for _, e := range plugins {
+		// a weight of zero is not permitted, plugins can be disabled explicitly
+		// when configured.
+		f.scorePluginWeight[e.Name] = int(e.Weight)
+		if f.scorePluginWeight[e.Name] == 0 {
+			f.scorePluginWeight[e.Name] = 1
+		}
+
+		// Checks totalPriority against MaxTotalScore to avoid overflow
+		if int64(f.scorePluginWeight[e.Name])*framework.MaxNodeScore > framework.MaxTotalScore-totalPriority {
+			return fmt.Errorf("total score of Score plugins could overflow"), 0
+		}
+		totalPriority += int64(f.scorePluginWeight[e.Name]) * framework.MaxNodeScore
+	}
+	return nil, totalPriority
+}
+
+func (f *frameworkImpl) expandMultiPointPlugins(profile *config.KubeSchedulerProfile, pluginsMap map[string]framework.Plugin) error {
+	// initialize MultiPoint plugins
+	for _, e := range f.getExtensionPoints(profile.Plugins) {
+		plugins := reflect.ValueOf(e.slicePtr).Elem()
+		pluginType := plugins.Type().Elem()
+		// build enabledSet of plugins already registered via normal extension points
+		// to check double registration
+		enabledSet := sets.NewString()
+		for _, plugin := range e.plugins.Enabled {
+			enabledSet.Insert(plugin.Name)
+		}
+
+		// track plugins enabled via multipoint separately from those enabled by specific extensions,
+		// so that we can distinguish between double-registration and explicit overrides
+		multiPointEnabled := sets.NewString()
+
+		disabledPlugins := sets.NewString()
+		for _, disabledPlugin := range e.plugins.Disabled {
+			disabledPlugins.Insert(disabledPlugin.Name)
+		}
+
+		for _, ep := range profile.Plugins.MultiPoint.Enabled {
```

Comment:
I added some cases to cover this, but it is essentially the same logic as this would have applied with `"*"` for default plugins in the old config. That is, specifically enabling a plugin for any extension point will enable it (including MultiPoint). 

We use the same function `mergePlugins()` which handles `"*"` for each extension point to disable all default plugins for that point, but after that it merges in any explicitly enabled plugins. This is the existing behavior